scripts/test/Muon/seq_fitting_tab_widget/seq_fitting_tab_presenter_test.py

```diff
@@ -204,6 +205,22 @@ def test_workspace_deleted_in_ads_updates_fit_table(self):
         self.model.get_runs_groups_and_pairs_for_fits.assert_called_once()
         self.view.fit_table.set_fit_workspaces.assert_called_once()
 
+    def test_that_disable_observer_calls_on_view_when_triggered(self):
+        disable_notifier = GenericObservable()
```

Comment:
see above

Reply:
The view in the setup has already been mocked out unlike in phase table so does not have the right methods.